chapters/5-processing.tex

```diff
@@ -63,10 +63,12 @@ \section{Making data ``tidy''}
  may be ambiguous in many raw datasets.
 This section will present what we call a \textit{tidy} data format,
 which is, in our experience, the ideal format to handle tabular data.
-Although this may at first seem out of place, as it comes before quality assurance,
-we believe that this is the appropriate first step in data cleaning,
-that it should be done continuously as data is received,
-and that quality assurance is best done using tidied data.
+Although data quality assurance and data tidying are often done at the same time,
+particularly one it comes to guaranteeing that observations are uniquely identified,
+we will treat it as the first step in data cleaning.
+This is because quality assurance can only be finalized using tidied data.
+In practice, both tidying and quality monitoring should be done continuously as data is received.
+
```

Comment:
```suggestion
We will treat tiding data as the first step in data cleaning,
even though in practice, 
both tidying and quality monitoring should be done simultaneously as data is received.
This is because quality assurance can only be finalized using tidied data,
when it is guaranteed that each observations are uniquely identified.
```
I am not trying to change what you are saying, but I had to read your first sentence a few times before I got what you were saying. I think this gets to the point you are trying to make quicker.